stjude/proteinpaint
# Improve the matrix sorting options to easily toggle sorting

## Description

… by cnv and/or consequence.

![Screenshot 2024-03-29 at 6 34 51 PM](https://github.com/stjude/proteinpaint/assets/411031/e3115564-7cc3-4b21-81f5-1ec0c37f7a37)

@congyu-lu Please check this branch in the neuroonc datasets

To test:
1. http://localhost:3000/testrun.html?dir=mass&name=matrix.integration
2. http://localhost:3000/testrun.html?dir=plots&name=matrix.sort.unit
3. GDC:
- Load http://localhost:3000/example.gdc.matrix.html?maxGenes=3&cohort=Gliomas
- `Cases -> Sort by values` should not have a `CNV` checkbox, when the `CNV` button is crossed out
- Click on the CNV button to show all CNV, then the `Cases -> Sort by values : CNV` checkbox should appear in the Cases menu
- Clicking on the `Cases -> Sort by values` CNV and Consequence checkboxes should intuitively re-sort the case columns 

## Checklist

[Check each task](https://github.com/stjude/proteinpaint/wiki/Pull-Request-Checklist) that has been performed or verified to be not applicable.
